src/org/labkey/test/TestScrubber.java

```diff
@@ -85,16 +88,19 @@ public void cleanSiteSettings()
 
         try
         {
-            ConfigureDbLoginPage.resetDbLoginConfig(this);
+            DatabaseAuthConfigureDialog.resetDbLoginConfig(this);
         }
         catch (RuntimeException e)
         {
             TestLogger.error("Failed to reset DB login config after test", e);
         }
 
         try
-        {
-            disableSecondaryAuthentication();
+        {   // disable any/all secondary auth configurations
+            LoginConfigurePage.beginAt(this).getSecondaryConfigurations()
+                .stream().filter(a -> a.getProvider().equals("TestSecondary"))
+                    .collect(Collectors.toList())
+                    .forEach(LoginConfigRow::clickDelete);
```

Comment:
Adam mentioned deleting the secondary auth configurations via API. Did that not pan out?

Reply:
We did implement that via API for the particular case of deleting a secondary login config discovered during login; because that one occurs on a URL containing the config ID it was trivial

For the nonce this seemed easier to do via the UI